src/components/Search/SearchResultsListItem.tsx

```diff
@@ -66,7 +72,19 @@ function SearchResultsListItem({
   const ListItemComponent = listItemComponent === 'radio' ? Radio : Checkbox;
   const revisionHash = truncateHash(item.revision);
   const commitMessage = getLatestCommitMessage(item);
-  const itemDate = new Date(item.push_timestamp * 1000);
+
+  const itemDate = dayjs.unix(item.push_timestamp);
+
+  const date = itemDate.format('MM/DD/YY');
+  const localTime = itemDate.format('HH:mm');
```

Comment:
similarly here, it would be better to have just 1 digit when possible, therefore you can use `H:m`.